sysutils/wormhole-william/Portfile

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/psanford/wormhole-william v1.0.4
```

Comment:
`go.setup` is of the form `<package name> <version> <version-prefix> <version-suffix>`

So `v1.0.4` needs to become `1.0.4 v`, as in: `github.com/psanford/wormhole-william 1.0.4 v`

`go2port` needs to be updated to do the right thing here.

Reply:
I changed it to `1.0.4 v`.
I noticed you already have pull request open for `go2port` so that's already taken care of.

Reply:
OK thank you @jasperweiss - yea, there's still some more things to clear up for that PR, but hopefully we'll get that sorted.  Thanks for this submission.